crates/bevy_mesh/src/primitives/dim2.rs

```diff
@@ -726,6 +751,15 @@ pub struct RhombusMeshBuilder {
     half_diagonals: Vec2,
 }
 
+impl RhombusMeshBuilder {
+    /// Creates a new [`RhombusMeshBuilder`] from a horizontal and vertical diagonal size.
+    pub const fn new(horizontal_diagonal: f32, vertical_diagonal: f32) -> Self {
```

Comment:
Not sure, but I think you may want a check for positive values here, using negative values will most likely give you the wrong winding order on the resulting mesh.

Reply:
After testing, negative values made the mesh disappear. I'll add a `debug_assert!` warning against it.